media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -42,9 +44,10 @@ BDEPEND="
 "
 RDEPEND="
 	media-libs/alsa-lib
-	sys-apps/dbus
+	sys-apps/dbus[${MULTILIB_USEDEP}]
 	sys-libs/ncurses[unicode]
-	virtual/libudev
+	virtual/libintl[${MULTILIB_USEDEP}]
+	virtual/libudev[${MULTILIB_USEDEP}]
 	bluetooth? (
 		aac? ( media-libs/fdk-aac )
```

Comment:
No multilib needed for fdk-aac?

Reply:
Non-native ABIs have completely disabled Bluetooth, since we only need the jack libraries and, if pipewire-alsa is enabled, also the ALSA plugin, none of which need AAC or any other audio codec. I'm not entirely sure about future but I doubt libraries directly host SPA plugins, so unless things are re-architected, this will stay true even if applications start to directly use PipeWire rather than JACK/ALSA/PA APIs.